Unable to compile the C++ generated code using -e switch

I run the following command to generate standalone CPP code
```
pythran -e sum_primes.py
```
Then I tried to compile it
```
pythran  -v sum_primes.cpp
INFO          sys file exists: /usr/local/lib/python3.8/dist-packages/pythran/pythran.cfg
INFO     platform file exists: /usr/local/lib/python3.8/dist-packages/pythran/pythran-linux.cfg
WARNING      user file does not exist: /home/run/.pythranrc
INFO     pythranrc section [pythran] is valid and options are correct
INFO     pythranrc section [typing] is valid and options are correct
INFO     pythranrc section [compiler] is valid and options are correct
running build_ext
new_compiler returns <class 'distutils.unixccompiler.UnixCCompiler'>
building 'sum_primes' extension
C compiler: x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC

compile options: '-DENABLE_PYTHON_MODULE -D__PYTHRAN__=3 -DPYTHRAN_BLAS_BLAS -I/usr/local/lib/python3.8/dist-packages/pythran -I/usr/local/lib/python3.8/dist-packages/numpy/core/include -I/usr/local/include -I/usr/include -I/usr/include/python3.8 -c'
extra options: '-std=c++11 -fno-math-errno -w -fvisibility=hidden -fno-wrapv'
x86_64-linux-gnu-gcc: sum_primes.cpp
In file included from sum_primes.cpp:32:
/usr/local/lib/python3.8/dist-packages/pythran/pythonic/builtins/range.hpp: In member function ‘bool {anonymous}::pythonic::builtins::{anonymous}::range_iterator::operator<(const {anonymous}::pythonic::builtins::{anonymous}::range_iterator&) const’:
/usr/local/lib/python3.8/dist-packages/pythran/pythonic/builtins/range.hpp:67:54: error: ‘CHAR_BIT’ was not declared in this scope
   67 |     const long sign = +1 | (step_ >> (sizeof(long) * CHAR_BIT - 1));
      |                                                      ^~~~~~~~
In file included from /usr/local/lib/python3.8/dist-packages/pythran/pythonic/builtins/range.hpp:6,
                 from sum_primes.cpp:32:
/usr/local/lib/python3.8/dist-packages/pythran/pythonic/builtins/range.hpp:1:1: note: ‘CHAR_BIT’ is defined in header ‘<climits>’; did you forget to ‘#include <climits>’?
  +++ |+#include <climits>
    1 | #ifndef PYTHONIC_BUILTIN_RANGE_HPP
CRITICAL Cover me Jack. Jack? Jaaaaack!!!!
E: error: Command "x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DENABLE_PYTHON_MODULE -D__PYTHRAN__=3 -DPYTHRAN_BLAS_BLAS -I/usr/local/lib/python3.8/dist-packages/pythran -I/usr/local/lib/python3.8/dist-packages/numpy/core/include -I/usr/local/include -I/usr/include -I/usr/include/python3.8 -c sum_primes.cpp -o /tmp/tmpr114inv5/sum_primes.o -std=c++11 -fno-math-errno -w -fvisibility=hidden -fno-wrapv" failed with exit status 1
```
But I am getting the above error. Is this a function or bug?

--- a/pythran/pythonic/builtins/range.hpp
+++ b/pythran/pythonic/builtins/range.hpp
@@ -3,6 +3,8 @@
 
 #include "pythonic/include/builtins/range.hpp"
 
+#include <climits>
+
 #include "pythonic/builtins/range.hpp"
 #include "pythonic/types/list.hpp"
 

